guides/traversing-pages.rst

```diff
@@ -81,6 +81,37 @@ that you can make same old traversing on such elements as well:
 
     // find some field INSIDE form with class="register"
     $field = $registerForm->findField('Email');
+    
+Webassert
```

Comment:
The assertion system should not be in ``traversing-pages`` IMO. It should be a dedicated chapter